src/index.js

```diff
@@ -1,28 +1,22 @@
 import './style.css';
 import render from './render.js';
+import { getUserScore, setUserScore } from './api.js';
 
-const arr = [
-  {
-    user: 'Name',
-    score: '100',
-  },
+const refreshBtn = document.querySelector('.refresh-btn');
 
-  {
-    user: 'Name',
-    score: '200',
+const form = document.querySelector('form');
 
-  },
+refreshBtn.addEventListener('click', async () => {
+  const result = await getUserScore();
+  render(result);
+});
 
-  {
-    user: 'Name',
-    score: '300',
-
-  },
-
-  {
-    user: 'Name',
-    score: '400',
-  },
-];
-
-render(arr);
+form.addEventListener('submit', async () => {
+  const { userName, userScore } = form.elements;
+  const game = {
+    user: userName.value,
+    score: userScore.value,
+  };
+  form.reset();
+  await setUserScore(game);
```

Comment:
- [ ] when trying to add a new score it doesn't actually add the new score, please try to fix this.

Reply:
For some reason, the project works well on firefox but don't know why I don't on my chrome browser

Reply:
Still my score does not appear when I add new score. Also for me it does not work on Firefox browser either. Your fetching data looks fine but I think the problem is when you send the new data which is not being saved on the server. Try to fix this issue

Reply:
Oh! Now I see where it's misbehaving... i forgot to add the `e.preventDefault()` to the form on submit... 

:+1: